introduction.md

```diff
@@ -0,0 +1,4 @@
+Hi i'm kevin hascoet i'm 21 soon 22 i'm learning on code school TheMrMelc .
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.